Tasks/Kurilina/Task_6/Task_favorite_6.py

```diff
@@ -0,0 +1,17 @@
+
+
+word = "FizzBuzz"
+print(*(word[(0,4,4)[x%3]:4 if x % 5 else 8] or x for x in range(1,101)))
```

Comment:
it is really interesting

Reply:
I think so too) but it was found on the internet